scripts/transpilation/copy-types.js

```diff
@@ -32,8 +32,7 @@ const typeFiles = execSync(
 typeFiles.map((typeFile) => {
   let component;
   const paths = typeFile.split('packages/');
-  const regEx = new RegExp('/');
-  const componentPath = paths[1].split(regEx);
+  const componentPath = paths[1].split('/');
```

Comment:
ESLint was complaining about using the RegExp constructor instead of using a regex directly